docs/topics/assembly_messaging-work-queue-mission-booster.adoc

```diff
@@ -0,0 +1,46 @@
+[id='mission-messaging-work-queue-{parameter-runtime}']
+= {name-mission-messaging-work-queue} Mission - {runtime} Booster
+
+include::messaging-work-queue-mission-intro-paragraph.adoc[leveloffset=+1]
+
+include::con_about-messaging-work-queue.adoc[leveloffset=+1]
+
+This booster shows you how to:
+
+* Send work requests to a work queue shared by multiple worker processes.
+* Use the broker to receive and process incoming work requests.
+* Send the processed result back to the original process.
+* Scale the number of workers according to the current requirements of your application.
+* Monitor the status of the worker processes.
+
+// see if necessary, if so, create
+//include::con_messaging-work-queue-concepts.adoc[leveloffset=+1]
+
+//adapt for this mission
+include::con_messaging-work-queue-design-tradeoffs.adoc[leveloffset=+1]
+
+// keep or throw away? keep, probably
+include::proc_viewing-the-booster-source-code-and-readme.adoc[leveloffset=+1]
+
+// define these properly, where necesary
+:parameter-mission-name: {name-mission-messaging-work-queue}
+:parameter-mission: messaging-work-queue
+
+//is this going to be configured to use FMP? it should if it is to be in the launcher?
+include::assembly_deploying-the-booster-to-openshiftonline.adoc[leveloffset=+1]
+
+include::assembly_deploying-the-booster-to-openshiftlocal.adoc[leveloffset=+1]
+:parameter-mission!:
+
+include::proc_deploying-the-booster-to-openshiftcontainerplatform.adoc[leveloffset=+1]
+
+// create a new interaction section for this booster
+//include::health-check-mission-basic-interaction-{parameter-runtime}.adoc[leveloffset=+1]
+
+ifndef::node-js[]
+include::proc_running-the-booster-integration-tests.adoc[leveloffset=+1]
```

Comment:
Can you confirm this (`mvn clean verify -Popenshift,openshift-it`) will work?

Reply:
@Ladicek TBH, I haven't tested it yet and therefore can not confirm. Since the POM file does not contain configurations for the `openshift and `openshift-it` profiles, my guess is it would not work unless properly configured first:)